pyhomematic/devicetypes/actors.py

```diff
@@ -372,6 +372,22 @@ def ELEMENT(self):
         return [4]
 
 
+class IPKeySwitchLevel(IPSwitch, HMEvent, HelperActionPress, HelperActorLevel):
+    """
+    Switch turning plugged in device on or off and measuring energy consumption.
```

Comment:
You might want to update the description when this works, to specify that the switch has some status LEDs
```suggestion
    Switch turning plugged in device on or off, measuring energy consumption and having user-controlled status LED.
```